src/lib.rs

```diff
@@ -1440,6 +1440,12 @@ impl Node {
 	}
 }
 
+impl Drop for Node {
+	fn drop(&mut self) {
+		let _ = self.stop();
```

Comment:
If the node is already stopped, this will return `Error::NotRunning`. I guess that is ok though.